hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/impl/WipeFungiblePrecompile.java

```diff
@@ -66,4 +78,17 @@ public long getMinimumFeeInTinybars(Timestamp consensusTime) {
                 wipeOp, "`body` method should be called before `getMinimumFeeInTinybars`");
         return pricingUtils.getMinimumPriceInTinybars(WIPE_FUNGIBLE, consensusTime);
     }
+
+    public static WipeWrapper decodeWipe(
+            final Bytes input, final UnaryOperator<byte[]> aliasResolver) {
+        final Tuple decodedArguments =
+                decodeFunctionCall(input, WIPE_TOKEN_ACCOUNT_SELECTOR, WIPE_TOKEN_ACCOUNT_DECODER);
+
+        final var tokenID = convertAddressBytesToTokenID(decodedArguments.get(0));
+        final var accountID =
+                convertLeftPaddedAddressToAccountId(decodedArguments.get(1), aliasResolver);
+        final var fungibleAmount = (long) decodedArguments.get(2);
```

Comment:
nit: In some places the numeric part of the `decodedArguments` is being cast directly to long whereas in other it's firstly being cast to `BigInteger` and then has its `longValue()` function invoked. Personally I prefer the former and it would be nice to have it be consistent throughout, but that's a low priority nit.